doctests/dt-streams.js

```diff
@@ -0,0 +1,340 @@
+// EXAMPLE: streams_tutorial
+// HIDE_START
+import assert from 'assert';
+import {
+  createClient
+} from 'redis';
+
+const client = await createClient();
+await client.connect();
+// HIDE_END
+// REMOVE_START
+await client.flushDb();
+// REMOVE_END
+
+// STEP_START xAdd
+const res1 = await client.xAdd(
+  'race:france', '*', {
+    'rider': 'Castilla',
+    'speed': '30.2',
+    'position': '1',
+    'location_id': '1'
+  }
+);
+console.log(res1); // >>> 1700073067968-0
```

Comment:
without context of how comments here were done before, what is this implying, that it should be greater than that value?